python/hail/expr/functions.py

```diff
@@ -954,19 +954,19 @@ def parse_call(s):
     -----
     This method expects strings in the following format:
 
-    +--------+-------------------------+--------------+
-    | ploidy | Phased                  | Unphased     |
-    +========+=========================+==============+
-    |   0    | &#124;-                 |  -           |
-    +--------+-------------------------+--------------+
-    |   1    | &#124;i                 |  i           |
-    +--------+-------------------------+--------------+
-    |   2    | i&#124;j                |  i/j         |
-    +--------+-------------------------+--------------+
-    |   3    | i&#124;j&#124;k         |  i/j/k       |
-    +--------+-------------------------+--------------+
-    |   N    | i&#124;j&#124;...&#124;N| i/j/k/.../N  |
-    +------+------+-----------------------------------+
+    +--------+-------------+-------------+
+    | ploidy | Phased      | Unphased    |
+    +========+=============+=============+
+    |   0    | -           |  -          |
```

Comment:
There should be a pipe before the dash in the phased column for ploidy = 0. `|-`

Same for ploidy == 1. `|i`